Libplanet.Tests/Store/LiteDBStoreTest.cs

```diff
@@ -0,0 +1,336 @@
+using System;
+using System.Collections.Generic;
+using System.Collections.Immutable;
+using System.Security.Cryptography;
+using Libplanet.Action;
+using Libplanet.Blocks;
+using Libplanet.Tests.Common.Action;
+using Libplanet.Tx;
+using Xunit;
+
+namespace Libplanet.Tests.Store
+{
+    public class LiteDBStoreTest : IDisposable
```

Comment:
I believe we can write some black box tests (a kind of property-based tests?) that all `IStore` implementations should commonly pass, besides this kind of white block tests.  E.g.:

```csharp
store.PutBlock(block);
Assert.Equal(block, store.GetBlock(block.Hash));
```

Such tests also could help us to eliminate duplicated assertions between `FileStoreTest` and `LiteDBStoreTest`.